news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 27 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-07-29 15:30:00
+authors: [ AntoineThomas ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 1st to Sunday 07th of July 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+Last week was a very warm week, indeed. Did you find a solution to stay productive in your area?
```

Comment:
Why the `indeed` ? 😐

Reply:
this article will be published on next monday. and, this current week is very warm, indeed

Reply:
Isn't "indeed" supposed to be used when confirming someone's question ?

Like "it has been warm, right ? - yes indeed"

Reply:
ok